### PR TITLE
Validate number of org chart uploads

### PR DESCRIPTION
**Pull request by jaleszek**

### CODE REVIEW

File: app/models/form_answer_attachment.rb [hunk 1]
```diff
@@ -54,4 +55,17 @@ def store_file!
...
   end
+
+  private
+
+  def question_key_correctness
+    # possible improvements:
+    # - validating all upload questions in this way
+    # - smart collecting max-attachments information from the form structure
+    if question_key == "org_chart" && new_record?
+      if self.class.where(question_key: self.question_key).count > 0
```
Comment (matthewford) [inline app/models/form_answer_attachment.rb]: Redundant `self` detected.

File: app/views/qae_form/_upload_question.html.slim [hunk 2]
```diff
@@ -47,10 +47,10 @@
     span.button.button-add *possible_read_only_ops
       span
         | + Upload
-      input.fileinput-button.js-file-upload type="file" name="form[file]" id="q_#{question.key}" *possible_read_only_ops
+      input.fileinput-button.js-file-upload type="file" name="form[file]" id="q_#{question.key}" *possible_read_only_ops data-question-key=question.key
     - if question.links
       | &nbsp;
-      a.button.button-add.js-file-upload data-add-link="1" href="#" *possible_read_only_ops
+      a.button.button-add.js-file-upload data-add-link="1" href="#" *possible_read_only_ops data-question-key=question.key
```
Comment (RuslanHamidullin) [inline app/views/qae_form/_upload_question.html.slim]: We do not need this for links:
```
data-question-key=question.key
```

**Comment by RuslanHamidullin** [comment]
Looks ok, small note above ^

**Comment by jaleszek** [comment]
ok done

**Pull request merged**